app/scripts/constants.js

```diff
@@ -52,6 +52,10 @@ window.OPENSHIFT_CONSTANTS = {
   // true indicates that deployment metrics should be disabled on the web console overview
   DISABLE_OVERVIEW_METRICS: false,
 
+  // true indicates that none of the routers support wildcard subdomains and
+  // removes the option from the route creation form.
+  DISABLE_WILDCARD_SUBDOMAINS: false,
```

Comment:
@knobunc is it better to assume most routers will support this feature by default, or to assume most will not have it?